code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -475,6 +675,11 @@
 /datum/reagent/capsaicin/affect_ingest(var/mob/living/carbon/M, var/alien, var/removed)
 	if(alien == IS_DIONA)
 		return
+	if(alien == IS_ALRAUNE) // VOREStation Edit: It wouldn't affect plants that much.
```

Comment:
Isn't there a proc check like is_plant or something along those lines for species/mobs that applies to both diona and alraune? Addign a whole separate reagent tag for a single interaction seems a bit overkill.

Reply:
The reagent_tag is specifically for chemical stuff, the IS_PLANT does other things, specifically making mobs immune to electric shocks, immune to hallucinations, immune to infections, and handles special Xenobotany interactions. The IS_ALRAUNE and IS_DIONA are separate checks used for chemistry/reagent processing, which is why rather than fuck with all that, I just added the bitflag for ALRAUNE and added the check here.

Reply:
IS_DIONA has all sorts of bullshit that makes most medicines not work (indeed, most chemical reactions at all) hence why alraunes don't have it.

Reply:
And IS_ALRAUNE doesn't touch any of the stuff dionas do, it was specifically  added for capsaicin, if you'd prefer that removed, I can do that.

Reply:
honestly if you're gonna make capsaicin not affect certain species, then make an IS_AVIAN or IS_BIRD or whatever since that's literally how capsaicin works - then give it to nevreans